tests/hypothesis_strategies.py

```diff
@@ -0,0 +1,158 @@
+"""Hypothesis strategies."""
+import random
+
+import hypothesis.strategies as st
+import numpy as np
+from hypothesis import assume
+from hypothesis.extra.pandas import columns, data_frames
+
+from baybe.exceptions import NumericalUnderflowError
+from baybe.parameters.categorical import (
```

Comment:
Isn't it possible to import all of the parameters directly from `baybe.parameters`? Why do we need or want all of the groups here?

Reply:
Yes, it is. I've changed it for this PR for the sake of this comment, but we should really have an alignment how we want to handle imports within our own packages (which may be different from what users would do). If we decide this is what we prefer, dann we should probably change this in all files in a separate PR 👍🏼

Reply:
there was a case in a prev PR where importing from the high level caused a circular import problem and only importing from direct source worked so imo thats an argument to import from direct source for use within the package

Reply:
Ok, that works for me. Only in parts that are explicitly pointed towards users (user guides and examples), I'd propose to import stuff like a user would do.

Reply:
Ok, then let's keep the policy like that! I've dropped my commit accordingly